packages/hevea/hevea.2.36/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+maintainer: "Luc Maranget <Luc.Maranget@inria.fr>"
+authors: "Luc Maranget"
+homepage: "http://hevea.inria.fr/"
+bug-reports: "http://github.com/herd/hevea/issues/"
+doc: "http://hevea.inria.fr/doc/index.html"
+dev-repo: "git+https://github.com/maranget/hevea.git"
+license: ["QPL-1.0" "LGPL-2 with exceptions"]
+build: [make "PREFIX=%{prefix}%"]
+install: [make "PREFIX=%{prefix}%" "install"]
+post-messages: [
+  "The file 'hevea.sty' has been installed in %{lib}%/hevea but latex won't see it by itself" {success}
+]
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "ocamlbuild" {build}
+]
```

Comment:
That's fine, I was just saying that in case you wanted to fix that issue. In that case adding this conflict should work:
```suggestion
]
conflicts: [
  "ocaml-option-bytecode-only"
]
```